package.json

```diff
@@ -128,19 +128,18 @@
     "pretest": "pnpm run compile-tests && pnpm run compile && pnpm run lint",
     "lint": "eslint src --ext ts",
     "test": "node ./out/test/integration/runTest.js",
-    "unit:test": "jest",
-    "unit:test-watch": "jest --watch"
+    "unit:test": "mocha -r ts-node/register 'src/test/unit/**/*.spec.ts'"
   },
   "devDependencies": {
     "@babel/core": "^7.22.6",
     "@babel/preset-env": "^7.22.6",
     "@types/async-retry": "^1.4.5",
     "@types/glob": "^7.2.0",
-    "@types/jest": "^29.5.2",
     "@types/lodash": "^4.14.197",
     "@types/mocha": "^9.1.1",
+    "@types/mock-require": "^2.0.1",
     "@types/node": "16.x",
-    "@types/vscode": "^1.71.0",
+        "@types/vscode": "^1.71.0",
```

Comment:
Not really critical but please remove the whitespace here, did you change things here manually or was it just pressed by mistake when you were in that file?

Reply:
1040ce2